compiler/rustc_hir/src/hir.rs

```diff
@@ -4517,6 +4535,16 @@ pub enum ForeignItemKind<'hir> {
     Type,
 }
 
+impl ForeignItemKind<'_> {
+    pub fn descr(&self) -> &'static str {
+        match self {
+            ForeignItemKind::Fn(..) => "function",
+            ForeignItemKind::Static(..) => "static variable",
+            ForeignItemKind::Type => "type",
+        }
+    }
+}
```

Comment:
we could probably just use `tcx.def_descr`, but it causes a lot of diffs across the test suite because it calls associated functions `associated functions` or `methods` respectively